### ISSUE TITLE
[Public Collections] Cloning asset from a subscribed Public Collection is not possible

### ISSUE THREAD

**Issue by magicznyleszek** [kobotoolbox/kpi#2836]
## Description

It should be possible to clone an asset from a subscribed Public Collection. This asset should be cloned without parent (normally when you own a collection and clone an asset, the clone has the same parent).

Currently we get `500` error.

![clone-fail-pc-beta](https://user-images.githubusercontent.com/2521888/95882184-aecff400-0d79-11eb-8dde-1d6cd9bdcce6.gif)

**Comment by noliveleger** [comment]
@joshuaberetta, because it may touch the serializer and because I changed few things about the `parent` validation. I strongly recommenced that you start over #2839

**Comment by joshuaberetta** [comment]
Note: the issue is no longer a `500` error but a `400` after @noliveleger made changes noted in the previous comment, however the user is still unable to clone an asset from a public collection that's not owned by them.

After much fiddling with the backend, we finally figured out the issue lies on the frontend where `parent` is appended to the request payload sent to the api. This is fine if the user is performing the clone from their own collection, but causes an authentication error if done on a public collection - whether subscribed or not. With the `parent` attribute included, the clone tries to write to the collection and therefore fails.

The default behavior should be that the cloned asset ends up in the users root - whether in their `Library` or `Projects` depending on the asset type (essentially just preventing `parent` from being appended). A potential addition is to allow for the user to specify where they want the asset to be cloned to.

Current request payload:
```
clone_from=<uid>
name=<name of clone>
parent=http://<kpi url>/api/v2/assets/<parent uid>/
```

Correct request payload:
```
clone_from=<uid>
name=<name of clone>
```

**Comment by jnm** [comment]
Fixed by #2851
